examples/sensor-channels/src/components/SensorDataHandler.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from "react";
+
+import { SensorData } from "./SensorData";
+
+function getLocaleFormat(timestamp) {
+  const event = new Date(timestamp);
+  const locale = window.navigator.language;
+  return event.toLocaleString(locale);
+}
+
+class SensorDataHandler extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      sensors: {},
+      device: {
+        connection_type: false
+      },
+      connected: false,
+      alerts: []
+    };
+  }
+
+  setDeviceStatus = () => {
+    const { device, astarte } = this.props;
+    astarte.getDeviceDataById(device).then(response => {
+      this.setState({ device: { connection_type: response.connected } });
+    });
+  };
+
+  onInComingData = response => {
+    if (response.event === "new_event") {
+      const { sensors } = this.state;
+      const time = response.payload.timestamp;
+      const event = response.payload.event;
+      if (event.path) {
+        sensors[event.path] = {
+          value: event.value,
+          // event.path => '/${device_name}/value'
+          // name = event.path.split("/")[1]
+          name: event.path.split("/")[1],
```

Comment:
this expression is a bit cryptic.
Normally one would think it takes the second element (because indexes are 0 based) of the path, but if the path starts with `/` the returned array will contain an empty element, and the function will return the first group in the path.

Or add a comment explaining the meaning behind this one (it's meant to skip the first empty element returned considering the path starts with a `/`)